app/controllers/v0/sessions_controller.rb

```diff
@@ -93,5 +109,27 @@ def async_create_evss_account(user)
       auth_headers = EVSS::AuthHeaders.new(user).to_h
       EVSS::CreateUserAccountJob.perform_async(auth_headers)
     end
+
+    # :nocov:
+    def handle_completed_slo
+      logout_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse], saml_settings)
+
+      logger.info "LogoutResponse is: #{logout_response}"
+
+      if !logout_response.validate
+        logger.error 'The SAML Logout Response is invalid'
```

Comment:
I can't decide if this is `logger.error` worthy.  `.validate` returns false under 3 circumstances:

1. ID.me has something wrong with their config / certs ---> error worthy
2. We have something messed up with our config / certs ---> error worthy
3. User simply failed authentication ---> probably not error worthy

1 & 2 I'm sure of, #3 I dont really know if the user actually makes it back to us after failing auth at ID.me